astropy/io/fits/tests/test_image.py

```diff
@@ -73,7 +73,7 @@ def test_constructor_copies_header(self):
         # Original header should be unchanged
         assert phdr['FILENAME'] == 'labq01i3q_rawtag.fits'
 
-    @raises(ValueError)
+    @raises(IndexError)
```

Comment:
@pllim - oh sorry I thought I replied. But @MSeifert04 , I'm not sure to understand if your comment is about the `@raises` statement, which could be changed to a `with raises(...)` but is not so common, or about the `fits.open` discussion below ?